xarray/core/rolling_exp.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+
+from .pycompat import dask_array_type
+
+
+def _get_alpha(com=None, span=None, halflife=None, alpha=None):
+    # pandas defines in terms of com (converting to alpha in the algo)
+    # so use its function to get a com and then convert to alpha
+
+    com = _get_center_of_mass(com, span, halflife, alpha)
+    return 1 / (1 + com)
+
+
+def move_exp_nanmean(array, *, axis, alpha):
+    if isinstance(array, dask_array_type):
+        raise TypeError("rolling_exp is not currently support for dask arrays")
+    import numbagg
+    if axis == ():
+        return array.astype(np.float64)
+    else:
+        return numbagg.moving.move_exp_nanmean(
```

Comment:
Let's use the top level function `numbagg.move_exp_nanmean`, that seems more likely to be stable than the submodule `numbagg.moving`.